packages/stream_chat_persistence/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: stream_chat_persistence
+homepage: https://github.com/GetStream/stream-chat-flutter
+description: Official Stream Chat Persistence library. Build your own chat experience using Dart and Flutter.
+version: 1.0.0-rc
+repository: https://github.com/GetStream/stream-chat-flutter
+issue_tracker: https://github.com/GetStream/stream-chat-flutter/issues
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+
+dependencies:
+  moor: ^3.4.0
+  path: ^1.7.0
+  path_provider: ^1.6.27
+  sqlite3_flutter_libs: ^0.3.0
+  stream_chat:
```

Comment:
Consider using the current version defined in `stream_chat`. Melos will automatically link.